README.markdown

```diff
@@ -2,26 +2,659 @@
 
 [![Hackage](https://img.shields.io/hackage/v/criterion.svg)](https://hackage.haskell.org/package/criterion) [![Build Status](https://github.com/haskell/criterion/workflows/Haskell-CI/badge.svg)](https://github.com/haskell/criterion/actions?query=workflow%3AHaskell-CI)
 
-This package provides the Criterion module, a Haskell library for
-measuring and analysing software performance.
+`criterion` is a library that makes accurate microbenchmarking in
+Haskell easy.
 
-<a href="http://www.serpentine.com/criterion/fibber.html" target="_blank"><img src="www/fibber-screenshot.png"></a>
+<a href="https://hackage.haskell.org/package/criterion/src/www/fibber.html" target="_blank"><img src="https://hackage.haskell.org/package/criterion/src/www/fibber-screenshot.png"></a>
 
-To get started, read the [online
-tutorial](http://www.serpentine.com/criterion/tutorial.html), and take a look
+
+## Features
+
+* The simple API hides a lot of automation and details that you
+  shouldn't need to worry about.
+
+* Sophisticated, high-resolution analysis can accurately measure
+  operations that run in as little as a few hundred picoseconds.
+
+* [Output to active HTML](https://hackage.haskell.org/package/criterion/src/www/report.html) (with Javascript charts), CSV,
+  and JSON.  Write your own report templates to customize exactly how
+  your results are presented.
+
+* Linear regression model allows measurement of the effects of garbage
+  collection and other factors.
```

Comment:
Minor word-smithing suggestions:

```suggestion
* Sophisticated, high-resolution analysis which can accurately measure
  operations that run in as little as a few hundred picoseconds.

* [Output to active HTML](https://hackage.haskell.org/package/criterion/src/www/report.html) (with JavaScript charts), CSV,
  and JSON. Write your own report templates to customize exactly how
  your results are presented.

* Linear regression model that allows measuring the effects of garbage
  collection and other factors.
```

Reply:
By the way, I didn't write this text: I moved it here from some other files that were already in the repo.